### DISCUSSION TITLE
iframe location access

### DISCUSSION THREAD

**Discussion by ray007** [cefsharp/CefSharp#4192] [Q&A]
I have a webapp which uses an `iframe` (in a `dialog`) and tries to detect end of user workflow by checking the frame location from an event handler for the iframes load event.
This works fine in google chrome, but in CefSharp looking at `loadEvent.target.contentWindow.location` shows a strange object.

I assume, this happens because in my CefSharp app the js-app is local, and therefore the iframe is now a cross-origin frame.
Setting `sandbox` or not does not make a difference.

Is there anything I can do that will allow me access to the iframe's new location?
I already am disabling web-security in Cef settings...

**Comment by n303ef** [comment]
Such code helped me:
```
 settings.CefCommandLineArgs.Add("disable-features=IsolateOrigins");
 settings.CefCommandLineArgs.Add("disable-site-isolation-trials");
```

**Reply by ray007** [reply]
I've solved my problem for now with a redirect from the server, but this is looking good, thank you!
